geppetto-ui/src/components/flex-layout/style/light.css

```diff
@@ -139,7 +139,7 @@
 
 .flexlayout__tab_button:hover .flexlayout__tab_button_trailing,
 .flexlayout__tab_button--selected .flexlayout__tab_button_trailing{
-    background: transparent url("../images/close.png") no-repeat center;
+    background: transparent url("geppetto-ui/src/components/flex-layout/images/close.png") no-repeat center;
```

Comment:
shouldn't refer to source files. Webpack loaders ensure us proper reference in different contexts.